Automattic/wp-calypso
# Settings: add new performance tab

#### Changes proposed in this Pull Request

* **This applies to both Jetpack and wpcom simple sites**
* Adds new Performance tab to settings.
* Moves the following modules to this new tab:
  * `Performance & Speed` (Jetpack only, previously on Writing)
  * `Media`: Video and Lazy-load (JP 5.8-alpha up) (Jetpack only, previously on Writing)
  * `Jetpack Search` (previously on Traffic)
  * `Accelerated Mobile Pages (AMP)` (previously on Traffic)

#### Notes

* Initial issue: https://github.com/Automattic/jetpack/issues/10550
* PR for Jetpack: https://github.com/Automattic/jetpack/pull/10941
* This will need updated e2e tests.

#### Before

![image](https://user-images.githubusercontent.com/390760/50222065-fad4cc00-038e-11e9-90dc-9fe6e2efced0.png)


#### After - Jetpack free



![image](https://user-images.githubusercontent.com/390760/50647446-7b3f2380-0f70-11e9-9720-f3955dd784ae.png)


#### After - Jetpack Professional / Atomic



![image](https://user-images.githubusercontent.com/390760/50643623-8b520580-0f66-11e9-8b67-c4b6159346f3.png)



#### After - wpcom simple site

![image](https://user-images.githubusercontent.com/390760/50222158-4c7d5680-038f-11e9-9d91-eeb556882180.png)

#### Testing instructions

* Fire up this branch.
* Take a look at the settings:
  * Ensure you see the new Performance tab.
  * Confirm changing settings saves them successfully.
  * Check the other settings tabs and make sure the modules are not duplicated.

Fixes #29545
